### ISSUE TITLE
chroot: cannot change root directory to '/': Operation not permitted

### ISSUE THREAD

**Issue by yumianhuli2** [carla-simulator/carla-autoware#102]
Hello！

After  build the image with the following command:

cd carla-autoware && ./build.sh

I got 
REPOSITORY          TAG                   IMAGE ID            CREATED             SIZE
carla-autoware      latest                609bb5968338        13 hours ago        11GB

But Whether I run. /run.sh or sudo docker run -it carla -autoware:latest /bin/bash I get a prompt that 
chroot: cannot change root Directory to '/': Operation not permitted

How can I fix this bug?Thanks！

**Comment by joel-mb** [comment]
Hi @yumianhuli2 !
Can you try again? I just merged a PR fixing this. Please, remove the `carla-autoware` image and create it again using the `build` script.

**Comment by yumianhuli2** [comment]
Hello！Thank you for update!
But When I run build script, I often encounter the following GPG errors:
![2020-10-08 08-18-13屏幕截图](https://user-images.githubusercontent.com/39210837/95401199-30b9ba80-093f-11eb-936e-b1fdd8400818.png)

**Comment by yumianhuli2** [comment]
> Hello！Thank you for update!
> But When I run build script, I often encounter the following GPG errors:
> ![2020-10-08 08-18-13屏幕截图](https://user-images.githubusercontent.com/39210837/95401199-30b9ba80-093f-11eb-936e-b1fdd8400818.png)
> 
> How can I solve this problem?Thank you！



> Hi @yumianhuli2 !
> Can you try again? I just merged a PR fixing this. Please, remove the `carla-autoware` image and create it again using the `build` script.

Hello！Thank you for update!
But When I run build script, I often encounter the following GPG errors:
![2020-10-08 08-18-13屏幕截图](https://user-images.githubusercontent.com/39210837/95401199-30b9ba80-093f-11eb-936e-b1fdd8400818.png)

How can I solve this problem?Thank you！

**Comment by yumianhuli2** [comment]
> Hi @yumianhuli2 !
> Can you try again? I just merged a PR fixing this. Please, remove the `carla-autoware` image and create it again using the `build` script.

Hello！Thank you for update!
But When I run build script, I often encounter the following GPG errors:
![2020-10-08 08-18-13屏幕截图](https://user-images.githubusercontent.com/39210837/95401199-30b9ba80-093f-11eb-936e-b1fdd8400818.png)

How can I solve this problem?Thank you！

**Comment by joel-mb** [comment]
Hi @yumianhuli2. Is this happening to you always? I cannot reproduce the error.

**Comment by yumianhuli2** [comment]
> Hi @yumianhuli2. Is this happening to you always? I cannot reproduce the error.

Yes,Only succeeded once, others failed dozens of times at least.

**Comment by yumianhuli2** [comment]
> Hi @yumianhuli2. Is this happening to you always? I cannot reproduce the error.

It works today,Thank you.
By the way, After I ran roslaunch carla_autoware_agent carla_autoware_agent.launch town:=Town01,I got below on server
side:
![Screenshot from 2020-10-09 14-20-16](https://user-images.githubusercontent.com/39210837/95549947-b706f700-0a3a-11eb-862b-e1afcd706d2b.png)

got below on autoware side:
![Uploading Screenshot from 2020-10-09 14-23-52.png…]()

**Comment by joel-mb** [comment]
Hi @yumianhuli2! Are you running the CARLA server and the autoware docker image on the same machine? I think this is not related to the GPU memory.

Can you please send me the complete log by running the following command:
```
roslaunch carla_autoware_agent carla_autoware_agent.launch town:=Town01 > log.txt
```
This will generate a `log.txt` inside the container. To copy the file to your host machine run:

```
docker cp <CONTAINER_ID>:/home/autoware/log.txt .
``` 
You can get the `<CONTAINER_ID>` by running `docker ps`

Thanks!

**Comment by yumianhuli2** [comment]
> Hi @yumianhuli2! Are you running the CARLA server and the autoware docker image on the same machine? I think this is not related to the GPU memory.
> 
> Can you please send me the complete log by running the following command:
> 
> ```
> roslaunch carla_autoware_agent carla_autoware_agent.launch town:=Town01 > log.txt
> ```
> 
> This will generate a `log.txt` inside the container. To copy the file to your host machine run:
> 
> ```
> docker cp <CONTAINER_ID>:/home/autoware/log.txt .
> ```
> 
> You can get the `<CONTAINER_ID>` by running `docker ps`
> 
> Thanks!

[log.txt](https://github.com/carla-simulator/carla-autoware/files/5507466/log.txt)
